pkg/relayer/proxy/sync.go

```diff
@@ -375,6 +389,48 @@ func (server *relayMinerHTTPServer) serveSyncRequest(
 	return relayRequest, nil
 }
 
+// serviceConfigTypeDefault indicates that the service config being used is
+// the default service config, as opposed to an RPC-type specific config.
+const serviceConfigTypeDefault = "DEFAULT_SERVICE_CONFIG"
+
+// getServiceConfig returns the service config for the service.
+// This will use either the RPC type specific service config or the default service config.
+func getServiceConfig(
+	supplierConfig *config.RelayMinerSupplierConfig,
+	request *http.Request,
+) (
+	serviceConfig *config.RelayMinerSupplierServiceConfig,
+	serviceConfigTypeLog string,
+	err error,
+) {
+	// If the following are true:
+	// 	- The RPC type is set for the service
+	// 	- The RPC type specific service config is available
+	// Then, use the RPC type specific service config.
```

Comment:
```suggestion
	// If the following are true:
	// 	- The RPC-type is set for the service
	// 	- The RPC-type service-specific config is available
	// Then, use the RPC-type service-specific config.
	// Otherwise, use the default service config.
```